doc/built-in-logs-pull.md

```diff
@@ -0,0 +1,152 @@
+# Built-in logs collation and upload capability
+
+IoT Edge supports native collation of module logs and upload to Azure Blob Storage. Users can access this functionality via direct method calls to the Edge Agent module. The following methods are available in support of this:
+
+- UploadLogs
+- GetTaskStatus
+
+## Feature enabling
+
+As of [1.0.8 release](https://github.com/Azure/azure-iotedge/releases/tag/1.0.8) of IoT Edge, this capability is available as an experimental feature. To enable it, the following environment variables need to be set for the edgeAgent (make note of the double underscores):
+
+| Environment Variable Name                | value  |
+|------------------------------------------|--------|
+| `ExperimentalFeatures__Enabled`          | `true` |
+| `ExperimentalFeatures__EnableUploadLogs` | `true` |
+
+## Recommended logging format
+
+For best compatibility with this feature, the recommended logging format is:
+
+```
+{LogLevel}{Timestamp}{MessageText}
```

Comment:
<{LogLevel}> {Timestamp} {MessageText}
Note the <> brackets for LogLevel and spaces between the 3 parameters.